### ISSUE TITLE
incomplete npm uploads

### ISSUE THREAD

**Issue by gdamjan** [PaulMaly/svelte-ts-preprocess#14]
it seems that both 1.1.0 and 1.1.1 were not uploaded properly on npm
```
curl -s https://registry.npmjs.org/svelte-ts-preprocess/-/svelte-ts-preprocess-1.1.1.tgz | tar tz
package/package.json
package/LICENSE
package/README.md
```

they're both missing the code (`package/src/index.js`)

**Comment by pyoner** [comment]
Hi, @gdamjan 
I think @PaulMaly will fix today
Thanks!

**Comment by PaulMaly** [comment]
Should be fixed in 1.1.2. Thanks!
